### PR TITLE
Document standardization

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Yii Queue Change Log
+# Yii Queue Extension Change Log
```
Comment (vjik) [inline CHANGELOG.md:1]: Better call package as "Yii Queue" without pistfix "extension".

File: README.md [hunk 2]
```diff
@@ -19,23 +15,19 @@ Documentation is at [docs/guide/README.md](docs/guide/README.md).
...
 [![type-coverage](https://shepherd.dev/github/yiisoft/queue/coverage.svg)](https://shepherd.dev/github/yiisoft/queue)
 
-## Installation
+An extension for running tasks asynchronously via queues.
 
-The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+## Requirements
 
-Either run
+- PHP 8.0 or higher.
```
Comment (vjik) [inline README.md:22]:
```suggestion
- PHP 8.1 or higher.
```

File: docs/guide/pt-BR/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Yii Queue extension
+
+An extension for running tasks asynchronously via queues.
```
Comment (vjik) [inline docs/guide/pt-BR/README.md:3]: Translate documentation or remove translation.

**Review by vjik — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (luizcmarin) [reply CHANGELOG.md:1]: "pistfix"... russian?

Comment (vjik) [reply CHANGELOG.md]: Postfix. I mean rename "Yii Queue Extension" to "Yii Queue".

Comment (vjik) [reply CHANGELOG.md]: It's not resolved.

Comment (luizcmarin) [reply CHANGELOG.md]: resolved in https://github.com/yiisoft/queue/pull/199/commits/a6392e7f84c379aeb6e91907d8bb56dcc7f2d2b3

Comment (luizcmarin) [reply CHANGELOG.md]: ok. done.

File: UPGRADE.md [hunk 4]
```diff
@@ -1,6 +1,3 @@
-Upgrading Instructions
-======================
-
-This file contains the upgrade notes. These notes highlight changes that could break your
-application when you upgrade the package from one version to another.
+# Yii Queue Upgrading Instructions
```
Comment (vjik) [inline UPGRADE.md:1]:
```suggestion
# Yii Queue Upgrading Instructions

This file contains the upgrade notes. These notes highlight changes that could break your
application when you upgrade the package from one version to another.
```

**Review by vjik — APPROVED** [review]

**Pull request merged**